data/brands/amenity/fast_food.json

```diff
@@ -185,6 +185,19 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Bagel Corner",
```

Comment:
Isn't that more of a Bakery ?
The results from overpass show that it's splti 
Rest looks good 👍

Reply:
Hi @Identitaet .
I've checked : the few nodes that have shop=bakery also have amenity=fast_food. A company (Digitalbricks) has added both tags together.
Bagel Corner works a bit like subway but with a bagel instead of bread.
They don't sell bread. which at least in France definitively exclude them from the bakery notation ;)
I'll send a message to Digitalbricks but I believe it's safe to integrate them as fast_food in name-suggestion-index

Reply:
okay if they don't sell bread I agree that's quite clear